testgrid/config/config.yaml

```diff
@@ -268,6 +268,8 @@ test_groups:
   gcs_prefix: kubernetes-jenkins/logs/kubernetes-e2e-gci-gce-scalability-release-1.4
 - name: kubernetes-e2e-gce-serial
   gcs_prefix: kubernetes-jenkins/logs/kubernetes-e2e-gce-serial
+- name: kubernetes-e2e-gce-disruptive
+  gcs_prefix: kubernetes-jenkins/logs/kubernetes-e2e-gce-disruptive
```

Comment:
eh, you'll need to add a dashboard-tab for the new test to pass the presubmit test, just search for where gce-serial is added